### ISSUE TITLE
[S] DUP: Update "Passes not required" text on homepage

### ISSUE THREAD

**Issue by MarshallHenson** [bcgov/parks-reso-public#406]
#### Description:

Change the "passes not required" text on the homepage to "Day-use pass not required" 
Change the "passes required" text on the homepage to "Day-use pass required" 

_Optional: add: "Overnight campsite reservation required" in a smaller/thinner font below. To discuss with UD_ (discussed during backlog refinement that we won't be adding this part)

#### Acceptance Criteria:
Given I am a user arriving to the DUP reservation site 
When I attempt to reserve a pass for a park that does not currently require a DUP 
Then I see the text "Day-use pass not required" next to the red dot 

Given I am a user arriving to the DUP reservation site 
When I attempt to reserve a pass for a park that requires a DUP 
Then I see the text "Day-use pass required" next to the green dot 

#### Development Checklist:

- [ ] Update DUP figma designs accordingly. Ensure the string of text doesn't overlap with the other text such as special closures notice.
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by Dianadec** [comment]
@MarshallHenson  to add another ticket for updating the DUP page wording away from 'pilot' https://reserve.bcparks.ca/dayuse/

**Comment by Dianadec** [comment]
@ndelraye this has a design task within it, fyi

**Comment by Christopher-walsh22** [comment]
I have updated the text as required. I believe we talked about how the design ticket was not required at a refinement.  I will reach out to @ndelraye  to find out the process for updating the Figma (as mentioned in the development checklist).

**Comment by ndelraye** [comment]
Figma has been updated

**Comment by manuji** [comment]
Tested on TEST: Passed
* Verified that the AC has been implemented. 
* Passing to PO review. 

Notes: 
* The optional requirement by Marshall has not been implemented.

**Comment by meyerdarcie** [comment]
Operating as intended, moving to done - this is good to go without the optional requirement for additional text.
